.markdownlint.json

```diff
@@ -0,0 +1,14 @@
+{
+    "no-inline-html": {
+        "allowed_elements": [
+            "details",
+            "center",
+            "p",
+            "i",
+            "label"
+        ]
+    },
+    "line-length": false,
+    "first-line-h1": false,
+    "siblings_only": true
```

Comment:
Most of the rule names are self explanatory but can you explain `sibling_only`? I trust your judgement here I am more just curious

Reply:
not sure if you addressed all violations or just on files your touched but if you want to look at/mark as done https://github.com/orgs/liatrio/projects/30/views/1?pane=issue&itemId=24041784 I would love it